src/main/java/com/shevtsov/view/AdminMenu.java

```diff
@@ -1,53 +1,241 @@
+        System.out.println("Input client id");
+        long clientID = MyUtilities.inputLong();
         System.out.println("Input name:");
-        String name = br.readLine();
+        String name = MyUtilities.inputString();
         System.out.println("Input surname:");
-        String surname = br.readLine();
+        String surname = MyUtilities.inputString();
         System.out.println("Input phone:");
-        String phone = br.readLine();
-        clientService.createClient(name, surname, phone);
+        String phone = MyUtilities.inputString();
+        if (clientService.modifyClient(clientID, name, surname, phone)) {
+            System.out.println("Client modified");
+        }
+    }
+
+    private void removeClient(ClientService clientService) {
+        System.out.println("Input client id");
+        long clientID = MyUtilities.inputLong();
+        if (clientService.removeClient(clientID)) {
+            System.out.println("Client removed");
+        }
+    }
+
+    private void addProduct(ProductService productService) {
+        System.out.println("Input name:");
+        String name = MyUtilities.inputString();
+        System.out.println("Input price:");
+        BigDecimal price = BigDecimal.valueOf(MyUtilities.inputLong());
+        if (productService.createProduct(name, price)) {
+            System.out.println("Product saved");
+        }
+    }
+
+    private void modifyProduct(ProductService productService) {
+        System.out.println("Input product id");
+        long productID = MyUtilities.inputLong();
+        System.out.println("Input new name:");
+        String newName = MyUtilities.inputString();
+        System.out.println("Input new price:");
+        BigDecimal newPrice = BigDecimal.valueOf(MyUtilities.inputLong());
+        if (productService.modifyProduct(productID, newName, newPrice)) {
+            System.out.println("Product modified");
+        }
     }
 
-    private void showMenu() {
-        System.out.println("1. Add client");
-        System.out.println("2. Modify client");
-        System.out.println("3. Remove client");
-        System.out.println("4. List all clients");
-        System.out.println("5. Add product");
-        //!!!подумать какие еще функции
+    private void removeProduct(ProductService productService) {
+        System.out.println("Input product id");
+        long productID = MyUtilities.inputLong();
+        if (productService.removeProduct(productID)) {
+            System.out.println("Product removed");
+        }
+    }
+
+    private void showOrder(OrderService orderService) {
+        System.out.println("Input order id:");
+        long orderID = MyUtilities.inputLong();
+        if (orderService.findOrderByID(orderID)) {
+            System.out.println("Order:");
+            System.out.println("...");
+            System.out.println("...");
+        } else {
+            System.out.println("There is no such order.");
+        }
     }
-}
+}
```

Comment:
OMG 241 line of code

Reply:
is it too much? will it be better to separate submenus in additional classes?

Reply:
yep